lib/engine/step/g_18_ga/track.rb

```diff
@@ -25,7 +25,8 @@ def process_lay_tile(action)
 
         def remaining_tile_lay?(entity)
           can_lay_tile?(entity) ||
-          (@game.p2_company.owner == entity && @game.abilities(@game.p2_company, :tile_lay)&.count&.positive?)
+            (@game.p2_company.owner == entity &&
+             @game.abilities(@game.p2_company, :tile_lay, time: 'Track')&.count&.positive?)
```

Comment:
Most likely this whole `remaining_tile_lay?` function can be removed once the relevant company has been changed to have `"when": "Track"` instead of `"when": ["SpecialTrack", "Track"]`